service-email/src/main/java/com/synopsys/integration/alert/service/email/model/EmailGlobalConfigModel.java

```diff
@@ -41,8 +43,8 @@ public EmailGlobalConfigModel obfuscate() {
         emailGlobalConfigModel.setSmtpUsername(smtpUsername);
         emailGlobalConfigModel.setAdditionalJavaMailProperties(additionalJavaMailProperties);
 
-        String maskedPassword = (smtpPassword != null) ? AlertRestConstants.MASKED_VALUE : null;
-        emailGlobalConfigModel.setSmtpPassword(maskedPassword);
+        emailGlobalConfigModel.setIsSmtpPasswordSet(StringUtils.isNotBlank(smtpPassword));
+        emailGlobalConfigModel.setSmtpPassword(null);
```

Comment:
Is the idea to remove (or perhaps rename?) the obfuscate method that is running this? If so, we would probably need to also refactor the ConfigurationCrudHelper and the Obfuscated interface

Reply:
Well it's still obfuscating by removing the sensitive data entirely i'd argue. We could change the name to something clearer